package.json

```diff
@@ -38,6 +38,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.0",
+    "chalk": "^1.0.0",
```

Comment:
In general I recommend hard coding the version number. Sometimes vendors don't follow the versioning guidelines and nothing forces them too, so if things change, it can break the build.